presto-spi/src/test/java/com/facebook/presto/spi/type/TestTypeSignature.java

```diff
@@ -35,7 +35,8 @@
 public class TestTypeSignature
```

Comment:
It seems like all the tests you've added use the row type.  But maybe add a test for a regular cast.